src/app/backend/resource/job/detail_test.go

```diff
@@ -100,3 +101,60 @@ func TestGetJobDetail(t *testing.T) {
 		}
 	}
 }
+
+func TestDeleteJob(t *testing.T) {
+	var jobCompletions int32
+	var parallelism int32
+
+	cases := []struct {
+		namespace, name string
+		expectedActions []string
+		job             *batch.Job
+	}{
+		{
+			"ns-1", "job-1",
+			[]string{"get", "list", "delete"},
+			createJob("job-1", "ns-1", map[string]string{"app": "test"}),
+		},
+		{
```

Comment:
Remove this case as it does not test what it's supposed to test. It does not test deletion of related pods. It is the same as first one essentialy.